### PR TITLE
Teamu 1239 shared UI component library

### PR DESCRIPTION
**Pull request by wtcarter**
Adds composite storybook and separate storybooks for older UIs to EOP

### CODE REVIEW

**Review by MartyP233 — COMMENTED** [review]
this is commiting a large number of files (300+) including some obfuscated javascript here - [packages/RainfallViewer/storybook-static/assets/index-DrFu-skq.js]. how trustworthy is it, and does it all need to be committed?

**Comment by wtcarter** [comment]
> this is commiting a large number of files (300+) including some obfuscated javascript here - [packages/RainfallViewer/storybook-static/assets/index-DrFu-skq.js]. how trustworthy is it, and does it all need to be committed?

Yeah, I was going to use those builds in Github if poss. but need to investigate further, so they can go.

File: packages/shared/ui/scripts/storybooks.ts [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+const startAllStorybooks = async () => {
+    const __filename = fileURLToPath(import.meta.url);
+    const __dirname = dirname(__filename);
+
+    const baseDir = join(__dirname, '../../../');
+    const storybookDirs = findStorybookDirs(baseDir);
+
+    console.log(`Found Storybooks in the following directories: ${storybookDirs.join(', ')}`);
+
+    let port = 6007;
```
Comment (MartyP233) [inline packages/shared/ui/scripts/storybooks.ts:52]: you ok with explicit port references and console logs being added here?

Comment (MartyP233) [reply packages/shared/ui/scripts/storybooks.ts:52]: there's a couple of other port references in other files too

Comment (wtcarter) [reply packages/shared/ui/scripts/storybooks.ts:52]: Yep, used for composite storybook

**Review by MartyP233 — APPROVED** [review]

**Pull request merged**